railties/lib/rails/generators/app_base.rb

```diff
@@ -409,8 +409,10 @@ def run_bundle
 
       def run_webpack
         if webpack_install?
-          rails_command "webpacker:install"
-          rails_command "webpacker:install:#{options[:webpack]}" if options[:webpack] && options[:webpack] != "webpack"
+          rails_command "webpacker:install", inline: true
+          if options[:webpack] && options[:webpack] != "webpack"
+            rails_command "webpacker:install:#{options[:webpack]}", inline: true
+          end
```

Comment:
These lines were hit by the tests I ran (I added logging to verify), but made no difference in test run time.

Reply:
...Because these specifc calls to `rails_command` are stubbed when tested: 

https://github.com/rails/rails/blob/774514659b823dd2374dce93e9f0ba2073aa3731/railties/test/generators/app_generator_test.rb#L903-L924

That being the case, perhaps it's not safe to add `inline: true` here, since it isn't really being exercised?